en/docs/guides/identity-lifecycles/outbound-provisioning.md

```diff
@@ -1,6 +1,6 @@
 # Outbound Provisioning
 
-This topic provides instructions on how to provision users to a trusted identity provider from the WSO2 Identity Server. A trusted identity provider is basically an identity provider that supports inbound provisioning such as Google, Salesforce, another Identity Server, etc. For more information, see [Outbound provisioning](insert-concepts-link).
+This topic provides instructions on how to provision users to a trusted identity provider from the WSO2 Identity Server. A trusted identity provider is basically an identity provider that supports inbound provisioning such as Google, Salesforce, another Identity Server, etc. For more information, see [Outbound provisioning](../../../concepts/provisioning-framework/#outbound-provisioning).
```

Comment:
"identity provider that supports inbound provisioning such as Google, Salesforce, another Identity Server, etc."

This is not clear. Gives an impression that the examples mentioned define what an inbound provisioning means. WDYT?